### PR TITLE
Fix GitHub container failure with latest Fedora Rawhide

### PR DESCRIPTION
**Pull request by martinpitt**
Turns out that the weird "m4 not found" error that I was brooding over yesterday was in fact not specific to Fedora ELN, but affects Rawhide as well. [Last night's container refresh broke due to this](https://github.com/rhinstaller/anaconda/actions/runs/358720903), and the RPM tests (which don't use a static container) now [fail as well](https://github.com/rhinstaller/anaconda/pull/2990/checks?check_run_id=1386774946).

This PR avoids the error in a more generic fashion than I did in #2987 (and I'll rebase/adjust that PR on top of this).

This is a bit urgent, as currently our CI is broken, so I'd like to skip the 24 hour thing for this. @jkonecny12 , @poncovka , can you please review this quickly?

### CODE REVIEW

**Comment by martinpitt** [comment]
I also took the liberty to robustify/simplify the container-autoupdate workflow. That will also make the [actions output](https://github.com/rhinstaller/anaconda/runs/1387956306?check_suite_focus=true) easier to read -- it's weird that the "Run tests in anaconda container" step appears as successful when it in fact failed.

**Comment by martinpitt** [comment]
[Successful run on my fork](https://github.com/martinpitt/anaconda/actions/runs/359226847). The tests appear failing here because of `pull_request_target`.

**Review by poncovka — APPROVED** [review]
Looks good to me. Thank you!

File: .github/workflows/container-autoupdate.yml [hunk 1]
```diff
@@ -39,12 +39,6 @@ jobs:
             tests/pylint/runpylint*.log
             tests/coverage-*.log
 
-      - name: Fail if local test did not worked
-        if: ${{ steps.run-tests.outcome == 'failure' }}
-        run: |
-          cat tests/test-suite.log
```
Comment (jkonecny12) [inline .github/workflows/container-autoupdate.yml:45]: Don't we want to have this `cat` call as part of ``run-tests`` step?

**Review by jkonecny12 — APPROVED** [review]
LGTM! Just a nitpick below.

File: .github/workflows/container-autoupdate.yml (hunk 1 above)
Comment (martinpitt) [reply .github/workflows/container-autoupdate.yml:45]: Good point, thanks! Fixed.

**Review by jkonecny12 — APPROVED** [review]
Looks even better now!

**Comment by martinpitt** [comment]
[Successful result on my fork](https://github.com/martinpitt/anaconda/actions/runs/359531390) after adjusting the daily container refresh.

**Pull request merged**